packages/ui/src/app/pages/WorkingGroups/MyRoles/MyRole.tsx

```diff
@@ -135,7 +137,7 @@ export const MyRole = () => {
               },
             ]}
           />
-          <TokenValueStat title="Stake height" value={worker.stake} />
+          <StakeStat value={worker.stake} minStake={100000} />
```

Comment:
The min stake must come from the worker's opening.

This is another case for testing :) Let's have a role page test, where we can set up different scenarios for the role. For this PR at least two tests are needed:

1.  The stake is OK
2.  The stake is lower than requried